src/advancedOpenFile.ts

```diff
@@ -18,11 +24,8 @@ class FilePickItem implements QuickPickItem {
   constructor(relativePath: string, absolutePath: string, filetype: FileType) {
     this.relativePath = relativePath
     this.absolutePath = absolutePath
-    this.label = this.relativePath
+    this.label = `${icons[filetype]} ${this.relativePath}`,
```

Comment:
Please delete this comma.
Because of this, `prettier` outputs the following code...

```diff
     this.absolutePath = absolutePath
-    this.label = `${icons[filetype]} ${this.relativePath}`,
-    this.filetype = filetype
+    ;(this.label = `${icons[filetype]} ${this.relativePath}`), (this.filetype = filetype)
   }
```